group.yml

```diff
@@ -176,3 +176,22 @@ image_build_log_scanner:
   - ppc64le.log
   - task_failed.log
   - orchestrator.log
+
+# Images which are NOT to be synced
+blacklist:
+- ose-ansible-operator
+- ose-ansible-service-broker
```

Comment:
These are still not distgit names. The code you added to doozer will do nothing.